### PR TITLE
add block annotation

### PR DESCRIPTION
**Pull request by wodka**
I am targeting this branch, because it is a new annotations. 

## Changelog

```markdown
### Added
- added block annotation

### Fixed
- fixed id setting for admin annotation
```

## To do
- [x] Update the tests
- [x] Update the documentation
- [x] Add an upgrade note

## Subject

add block annotation

### CODE REVIEW

File: Annotation/Block.php [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    /**
+     * @param ClassMetadata $metadata
+     */
+    public function processMetadata(ClassMetadata $metadata)
+    {
+        $tag = array();
+
+        $tag = array_filter($tag, function ($v) {
+            return !is_null($v);
+        });
```
Comment (greg0ire) [inline Annotation/Block.php]: wat

File: Resources/doc/reference/annotations.rst [hunk 2]
```diff
@@ -71,6 +71,28 @@ All you have to do is include Sonata\AdminBundleAnnotations and define the value
     }
 
 
+
+Define Blocks
+^^^^^^^^^^^^^
+
+All you have to do is include Sonata\AdminBundleAnnotations and define the values you need.
```
Comment (greg0ire) [inline Resources/doc/reference/annotations.rst]: Please quote the FQCN with backticks and remove the "s"

File: Annotation/Block.php (hunk 1 above)
Comment (wodka) [inline Annotation/Block.php]: upps - removed it

File: Resources/doc/reference/annotations.rst (hunk 2 above)
Comment (wodka) [inline Resources/doc/reference/annotations.rst]: done

File: Resources/doc/reference/annotations.rst [hunk 3]
```diff
@@ -71,6 +71,28 @@ All you have to do is include Sonata\AdminBundleAnnotations and define the value
     }
 
 
+
+Define Blocks
+^^^^^^^^^^^^^
+
+All you have to do is include `Sonata\AdminBundleAnnotation` and define the values you need.
```
Comment (OskarStark) [inline Resources/doc/reference/annotations.rst]: Sonata\AdminBundleAnnotation

did you mean: Sonata\AdminBundle\Annotation?

**Comment by wodka** [comment]
@core23 you're right - I'm using sonataadmin all the time I forgot that a separate bundle exists for it. I will add it there

**Pull request closed without merging**